ansible_sdk/_util/metrics.py

```diff
@@ -203,5 +208,5 @@ async def collect_metrics(self, status_obj: AnsibleJobStatus):
 
         with tarfile.open(metrics_tar_filename, "w") as tfh:
             for filename in datafiles:
-                tfh.add(filename)
-                os.remove(filename)
+                tfh.add(os.path.basename(filename))
+                await aiofiles.os.remove(filename)
```

Comment:
We probably don't need the `aiofiles` dep for this- just wrap an async shim around `os.remove` .